### PR TITLE
docs: add non-python dependencies to installation instructions

### LINKED ISSUE #136: DOC: non-python dependencies
`git` and `tig` are dependencies.

### PR DESCRIPTION
**Pull request by loj**
Closes #136

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -10,10 +13,30 @@ environment. E.g. with ``venv``:
...
+
+Additional non-python dependencies need to be installed:
+
+**Debian/Ubuntu**:
+
+.. code:: shell
+
+   apt-get install git tig tree
+
+**macOs**:
```
Comment (aqw) [inline docs/source/installation.rst:38]: World's tiniest nit, but it slays me: macOs -> macOS

:-D

**Review by aqw — CHANGES_REQUESTED** [review]
I absolutely love the PR preview for RTD. This is /so/ cool.

File: docs/source/installation.rst (hunk 1 above)
Comment (loj) [reply docs/source/installation.rst]: Oops, thanks for catching that!

**Comment by aqw** [comment]
Thanks!

**Pull request merged**